pages/index.js

```diff
@@ -1,3 +1,21 @@
+import Head from "next/head";
+import Header from "../components/Header";
+// import Footer from "../components/Footer";
+
 export default function Home() {
-  return <h1>Willkommen zum 🐙-Capstone-Template!</h1>;
+  return (
+    <>
+      <Head>
+        <title>
+          Search Find Go! - Find Your Favourite Movies and Series in
+          Milliseconds
+        </title>
+        <meta name="description" content="Generated by create next app" />
```

Comment:
I guess maybe the `title` would be `Search Find Go!` and the `description` would be `find your favorite movies and series in milliseconds`. 
```suggestion
        <title>
          Search Find Go!
        </title>
        <meta name="description" content="Find Your Favourite Movies and Series in
          Milliseconds" />
```